src/sentry/static/sentry/app/stores/groupingStore.tsx

```diff
@@ -45,14 +45,14 @@ type State = {
 
 type ScoreMap = Record<string, number | null>;
 
-type Item = {
+export type Fingerprint = {
```

Comment:
I'm using this name from a couple of comments:
`<span>{t('Merged fingerprints with latest event')}</span>{' '}`
and
```
      // List of fingerprints that belong to issue
      mergedItems: [],
```

Reply:
What about `MergedItem`?